project/profiles/capacity4more/modules/c4m/content/c4m_content_project/c4m_content_project.block.inc

```diff
@@ -13,7 +13,7 @@ function _c4m_content_project_block_c4m_block_create_project() {
 
   if ($user->uid) {
     if (user_access('create project content')) {
-      $markup = l(t('Create a project'), 'node/add/project', array('attributes' => array('class' => array('btn', 'btn-default'))));
+      $markup = '<div class="c4m-group-visitor-cta">' . l(t('Create a project'), 'node/add/project', array('attributes' => array('class' => array('btn', 'btn-default')))) . '</div>';
```

Comment:
@guytepper 
It's better to add a specific class to the link and use the same styling